app/views/users/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_with model: @user, local: true do |f| %>
+  <%= render "shared/error_messages", object: @user %>
+
+  <%= f.label :name %>
+  <%= f.text_field :name, class: "form-control" %>
+
+  <%= f.label :email %>
```

Comment:
missing i18n for label

Reply:
Có anh ơi không phải mất đâu ạ. 
![screenshot from 2019-01-04 13-07-21](https://user-images.githubusercontent.com/45062663/50675596-b4839c00-1021-11e9-92f2-ed45307618fb.png)